simulation/manage.py

```diff
@@ -0,0 +1,96 @@
+import csv
+import argparse
+import time
+
+from web3 import Web3, HTTPProvider
+from solc import compile_files
+
+
+def get_token_instance(compiled_source):
+    with open('deploy_info.csv', 'rt') as text_file:
+        spamreader = csv.reader(text_file, quoting=csv.QUOTE_MINIMAL)
+        next(spamreader)
+        next(spamreader)
+        next(spamreader)
+        token_address = next(spamreader)[1]
```

Comment:
It would be better, I think, use here function get_csv_file_row() that you wrote.